src/tools/hooks/__tests__/useInitialFormParameters.spec.tsx

```diff
@@ -6,53 +6,106 @@ import axios from 'axios';
 import MockAdapter from 'axios-mock-adapter';
 
 import { LocationToPath, Location } from '../../../app/config/urls';
-import useInitialFormParameters from '../useInitialFormParameters';
+import useInitialFormParameters, {
+  FormValues,
+} from '../useInitialFormParameters';
 
-import defaultFormValues from '../../align/config/AlignFormData';
+import defaultAlignFormValues from '../../align/config/AlignFormData';
+import defaultPeptideSearchFormValues from '../../peptide-search/config/PeptideSearchFormData';
 import accessionsData from '../../../uniprotkb/components/entry/__tests__/__mocks__/accessionsData.json';
 
-describe('useInitialFormParameters', () => {
-  const customRenderHook = (path?: string, state?: unknown) => {
-    const history = createMemoryHistory();
-    if (path) {
-      history.push(path, state);
-    }
-    return renderHook(() => useInitialFormParameters(defaultFormValues), {
-      wrapper: ({ children }) => <Router history={history}>{children}</Router>,
-    });
-  };
+function customRenderHook<Fields extends string>(
+  defaultFormValues: Readonly<FormValues<Fields>>,
+  path?: string,
+  state?: unknown
+) {
+  const history = createMemoryHistory();
+  if (path) {
+    history.push(path, state);
+  }
+  return renderHook(() => useInitialFormParameters(defaultFormValues), {
+    wrapper: ({ children }) => <Router history={history}>{children}</Router>,
+  });
+}
```

Comment:
Should that be with, or near, the customRender helper? I'm wondering if there are other hooks that are tested in a similar way and that need to be wrapped in react-router's logic

Reply:
Good idea, moved this up with the other test helpers. Look OK?